PLAYBOOKS.md

```diff
@@ -0,0 +1,70 @@
+| `scraper.features.baseUrl`                         | Constant prefix part of the URL to be used by the routes                                                          | String       | Yes                                        | `"baseUrl": "https://aim-india.aai.aero/eaip-v2-02-2021"` |
+| `scraper.features.paths`                           | Collection of multiple routes the scrapper needs to take to reach the final chart page                            | Array        | Yes                                        | [See example](playbooks/IN.json)                          |
+| `scraper.features.paths / route`                   | Suffix path to be used along with `scraper.features.baseUrl` to visit pages                                       | String       | Yes                                        | `"route": "/index-en-GB.html"`                            |
+| `scraper.features.paths / navigations`             | CSS selector based navigation to be performed on the current `scraper.features.paths                              | route`       | Array                                      | Optional                                                  | [See example](playbooks/IN.json) |
+| `scraper.features.paths / navigations / selector`  | CSS Selector for the element that is to be used for navigation                                                    | String       | Yes                                        | `"selector": "frame[name=\"eAISNavigationBase\"]"`        |
+| `scraper.features.paths / navigations / attribute` | URL route giving attribute of the selected element                                                                | String       | Yes                                        | `"attribute": "src"`                                      |
+| `scraper.features.chart`                           | The CSS selector based configuration for getting and formatting the final chart URL                               | JSON         | Yes (can either use `chart` or `search`)   | [See example](playbooks/IN.json)                          |
+| `scraper.features.chart.baseUrl`                   | Constant prefix part of the URL to be used for all chart URLs                                                     | String       | Yes                                        | `"baseUrl": "${baseUrl}/eAIP/"`                           |
+| `scraper.features.chart.selector`                  | CSS Selector for the element that is to be used for getting the final chart URL                                   | String       | Yes (can either use `selector` or `xpath`) | `"selector": "a[title=\"${icao}\"]"`                      |
+| `scraper.features.chart.xpath`                     | XPath of the                                                                                                      | String       | Yes (can either use `selector` or `xpath`) | `"xpath": "//a[contains(text(),'${icao}')]"`              |
```

Comment:
Xpath of the ... missing text afterwards.

Reply:
Fixed